filter/config.go

```diff
@@ -32,14 +32,14 @@ func (c Config) Validate() error {
 	return nil
 }
 
-type CombinedFilter struct {
+type combinedFilter struct {
```

Comment:
Can this be marked as deprecated for one release and then made private? I know it's only used in mdatagen internally, but we can't be sure others out there aren't using this package that's 2 months old at this point

Reply:
I didn't pick up it was used anywhere in contrib. I'll follow your direction, sure.